testsuite/tests/kuadrant/authorino/conditions/section_conditions/test_authorization_condition.py

```diff
@@ -0,0 +1,46 @@
+"""Test condition to skip the authorization section of AuthConfig"""
+import pytest
+
+from testsuite.objects import Rule
+from testsuite.utils import extract_from_response
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, module_label):
+    """
+    Add to the AuthConfig:
+        - authorization with json rule on /post path, which will be used only on POST requests
+        - response with auth.authorization, to verify if authorization has been used
+    """
+    rule_post = Rule("context.request.http.path", "eq", "/post")
+    when_post = [Rule("context.request.http.method", "eq", "POST")]
+    authorization.authorization.auth_rule(module_label, rule_post, when=when_post)
+
+    authorization.responses.add(
```

Comment:
This is a last resort, ideally, you should devise a test case where you can figure the result out based on accepting/rejecting the request

Reply:
Reworked 👍